action-server/setup.cfg

```diff
@@ -0,0 +1,48 @@
+[tool:pytest]
+testpaths = tests
+norecursedirs = .git,.venv,venv
+xfail_strict = true
+addopts =
+    --strict
+    --cov actions
+    --cov-config setup.cfg
+    --cov-report term
+    --cov-report term-missing
+    --cov-fail-under=0
+
+[pytest-watch]
+ext = .py, .yaml, .cfg
+
+[coverage:run]
+branch = True
+omit = tests/*,.venv/*,venv/*
```

Comment:
`venv/*` is not necessary with out current setup (we only use `.venv`)

Reply:
There are a few more occurences below.

Reply:
CI/CD actually uses `venv`